src/core/network/client.ts

```diff
@@ -82,6 +82,13 @@ function includesMedia(payload: Record<string, unknown>) {
   })
 }
 
+function compactOptions<T>(options: T): T {
```

Comment:
Move `compactOptions` to `core/helpers` and fix other classes as well:

https://github.com/telegraf/telegraf/blob/81a1a40ae88d48895e87a5d7079b3b91f8c75191/src%2Ftelegraf.ts#L100-L104

Reply:
Moved `compactOptions`, and fixed `options` in `src/telegraf`.

Also modified compactOptions to allow `undefined` to passthrough so that we don't add `?? {}` all over the place. Banking on the fact that JS allows us to `{ ...x }` where `x` is undefined.